assets/css/style.css

```diff
@@ -0,0 +1,127 @@
+body {
+    background-image: url("../images/bananas.png")
+}
+
+.top-page {
+    position: relative;
+    bottom: 24px;
+    right: 16px;
+    width: 110%;
+    height: 60px;
+    background-color: #3be;
+    color: white;
+} 
+
+
+
+#subject {
+    font-size: 30px;
+    text-align: center;
+    border-bottom: 1px solid #3be;
+}
+
+#para-bio {
+    line-height: 40px;
+    font-family: 'Shadows Into Light', cursive;
+    position: relative;
+    left: 5px;
+    text-align: center;
+    
+}
+
+#about-head{
+    position: relative;
+    display: inherit;
+
+}
+
+#portfolio {
+    position: relative;
+    display: inherit;
+}
+
+#main-body, #index-photo {
+    float: left;
+}
+
+#contact-head {
+    position: relative;
+    display: inherit;
+
+}
+
+#name-head {
+    position: relative;
+    font-size: 36px;
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    left: 20px;
+}
+
+.header-links {
+    position: relative;
+    display: inline;
```

Comment:
just keep display inline here so that the ul will show up next to each other, if you do all these changes you'll have less complex and more centered header!